awx/main/migrations/_multi_cred.py

```diff
@@ -0,0 +1,12 @@
+def migrate_to_multi_cred(app, schema_editor):
+    Job = app.get_model('main', 'Job')
+    JobTemplate = app.get_model('main', 'JobTemplate')
+
+    for cls in (Job, JobTemplate):
+        for j in cls.objects.iterator():
+            j.credentials.add(j.credential)
```

Comment:
Note to self: I should check `if j.credential` here, since it's possible for this value to be `None` if "ask for cred" is turned on.